### DISCUSSION TITLE
Sending an image to a webhook for the document AI expenses template

### DISCUSSION THREAD

**Discussion by menashkes** [AntlerVC/firetable#1358] [Support / Q&A]
I am sorry but I can't figure out how to do it.

Please advise.

Thanks.

**Comment by hrutik7** [comment]
Hey @menashkes ,
could you please elaborate more on this please ?
so we can understand what's going on

**Comment by harinij** [comment]
Closing older issue, in case the issue still persists you can reopen by commenting with further detailed explanation including reproduction steps of the issue.
